standard/docs/en/implementation/milestones.md

```diff
@@ -0,0 +1,52 @@
+# Milestones
+
+Milestones can be included within the planning, tender, contract and contract implementation blocks. 
+
+* The planning milestone block may describe anticipated milestones during the planning stage, such as the preparation 
+of key studies. Usually using the 'preProcurement' milestone type code.
+
+* Tender milestone blocks should be used to describe anticipated milestones during the proposed delivery 
+of a contract, as well as key dates from the process of tendering and awarding the contract.
+
+* Contract milestone block should be used for events related to the signing of the contract documents.
+
+* Contract implementation milestones block should be used for any event related to delivery of the contract.
+
+The nature of the milestone is indicated by the milestone type code list.
+
+The lifecycle of the milestone should be modeled with the fields: `dueDate`, `dateMet`, `dateModified` and `status`.
```

Comment:
Helpful to make clear here these are from the [milestone status codelist](https://standard.open-contracting.org/latest/en/schema/codelists/#milestone-status) and provide a link to it.

Reply:
I think that for this example the 'code' field is not much relevant, but I added the link as you suggested!